### PR TITLE
AGP-151: A simple Network Budget

### PR DESCRIPTION
**Pull request by LouisGrx**
Here is the meta track AGP related to https://github.com/aragon/AGPs/pull/149/files?short_path=618aa76#diff-618aa76b7b989ee736b05e779fd24b83

### CODE REVIEW

File: AGPs/network-buget.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+| Title                   | Author                 | Status    | Created    |
```
Comment (john-light) [inline AGPs/network-buget.md]:
```suggestion
---
AGP: N/A
Title: A simple Network Budget
Author: Louis Giraux (@LouisGrx)
Status: Stage IV
Track: Finance
Created: 2019-11-20
---
```

File: AGPs/network-buget.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+|-------------------------|------------------------|-----------|------------|
```
Comment (john-light) [inline AGPs/network-buget.md]:
```suggestion
```

File: AGPs/network-buget.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| A simple Network Budget | Louis Giraux (@louisgrx) | Stage III | 2019-11-20 |
```
Comment (john-light) [inline AGPs/network-buget.md]:
```suggestion
```

File: AGPs/network-buget.md (hunk 1 above)
Comment (john-light) [reply AGPs/network-buget.md]: Please use the AGP header format provided in the templates directory.

Comment (LouisGrx) [reply AGPs/network-buget.md]: Happy to do so. Wouldn't display properly so used another one

**Comment by luisivan** [comment]
Withdrawing this AGP from ANV-5 on behalf of the Aragon Association

**Comment by john-light** [comment]
With the approval of [AGP-155](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-155.md) in Aragon Network Vote #6, the AGP process has now been sunset. All open AGP pull requests are now being closed.

Thank you for participating in the AGP process!

**Pull request closed without merging**